HyperionPublisher:
    ros__parameters:
        interrogator/ip_address: 10.0.0.5